compiler/semobjconstr.nim

```diff
@@ -365,18 +365,6 @@ proc semObjConstr(c: PContext, n: PNode, flags: TExprFlags): PNode =
   # branches will be reported as an error):
   let initResult = semConstructType(c, result, t, flags)
 
-  # It's possible that the object was not fully initialized while
```

Comment:
this definitely should show at least a summarizing comment; might as well keep this block IMO